Bloom.md

```diff
@@ -0,0 +1,341 @@
+This crate also has had a breaking change across versions and bloom filters of the older version are not loadable on the
+newer versions.
+
+
+### Large BloomFilter objects
+
+Create and Delete operations on large bloom filters take longer durations and will block the main thread for this duration. 
+Because of this, the following operations will be handled differently:
+* defrag callback: If the memory used by any bloom filter within the bloom object is greater than X MB (`bloom_large_item_threshold`
+    constant), we will skip defrag operations on this bloom object. Otherwise, we will defrag the bloom object in iterations
+    for each bloom filter in the bloom object.
+* free_effort callback: This callback decides the free effort for the bloom object. If it is greater than X MB
+    (`bloom_large_item_threshold` constant), we will return 0 to use async free on the bloom object.
+* create operations (BF.ADD/MADD/INSERT/RESERVE): We can compute the estimated size of the bloom filter that will be created as
+    a result of the operation and we can consider rejecting requests in case of unavailable memory. Scalable Bloom filters
+    will grow in used memory after creation - but only as a result of a BF.ADD, BF.MADD, or BF.INSERT operation and we can reject
+    these requests if we identify that it will result in a scale out operation that requires more memory than what is available.
+
+
+## Specification
+
+### Bloom Filter Command API
+
+The following Bloom Filter commands are supported and their API syntax is compatible with ReBloom:
+BF.ADD <key> <item>
+BF.EXISTS <key> <item>
+BF.MADD <key> <item> [<item> ...]
+BF.MEXISTS <key> <item> [<item> ...]
+BF.CARD <key>
+BF.INFO <key> [CAPACITY | SIZE | FILTERS | ITEMS | EXPANSION]
+BF.RESERVE <key> <false_positive_rate> <capacity> [EXPANSION <expansion>] | [NONSCALING]
+BF.INSERT <key> [CAPACITY <capacity>] [ERROR <fp_error>] [EXPANSION <expansion>] [NOCREATE] [NONSCALING] ITEMS <item> [<item> ...]
+
+Currently following commands (from ReBloom) are not supported:
+BF.LOADCHUNK <key> <iterator> <data>
+BF.SCANDUMP <key> <iterator>
+
+The BF.SCANDUMP command is used to perform an incremental save on specific Bloom filter object.
+The BF.LOADCHUNK is used to incrementally load / restore a bloom filter object from data from the BF.SCANDUMP command.
+
+The reason for not implementing these two commands is because the Module provides the ability to load and save BloomModule
+data type items during RDB load and save. BF.LOADCHUNK and BF.SCANDUMP are APIs to load bloom filter objects through commands,
+but since we will provide RDB save & load, having specific commands for the same purpose was not considered as required.
+
+Note: Based on the AOF Rewrite discussion, we will decide on whether these commands should be supported.
+
+### Configurations
+
+The default properties using which Bloom Filter objects are created can be controlled using configs.
```

Comment:
Could we also mention that these can be overridden for each bloom object via BF.RESERVE / BF.INSERT

Reply:
Yes, I updated the section